lib/pages/account.dart

```diff
@@ -0,0 +1,442 @@
+            actions: [
+              okButton,
+            ],
+          );
+
+          showDialog(context: context,
+            builder: (BuildContext context){
+              return alert;
+            },
+          );
+
+        } else {
+          Widget okButton = FlatButton(
+            child: Text("OK"),
+            onPressed: () {
+              Navigator.of(context).pop();
+            },
+          );
+
+          AlertDialog alert = AlertDialog(
+            title: Text("Settings"),
+            content: Text("Your settings have been saved successfully"),
+            actions: [
+              okButton,
+            ],
+          );
+
+          showDialog(context: context,
+          builder: (BuildContext context){
+            return alert;
+          },
+          );
+
+         // Navigator.of(context).pushReplacementNamed('/home');
+        }
+      });
+    }
+  }
+
+
+  @override
+  Widget build(BuildContext context) {
+
+    _firstNameController = TextEditingController(text: _account.firstName);
+    _lastNameController = TextEditingController(text: _account.lastName);
+    _phoneController = TextEditingController(text: _account.phoneNumber);
+
+    return Scaffold(
+      key: _scaffoldKey,
+      appBar: AppBar(
+        centerTitle: false,
+        title: Text('Account Settings'),
+        iconTheme: IconThemeData(
+          color: Colors.white
+        ),
+      ),
+      body: SingleChildScrollView(
+        padding: const EdgeInsets.all(16.0),
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: <Widget>[
+            Center(
+              child: Text(
+                "Email",
+                style: TextStyle(
+                  fontSize: 18,
+                  fontWeight: FontWeight.bold,
+                ),
+                textAlign: TextAlign.center,
+              ),
+            ),
+
+            Card(
+                shape: RoundedRectangleBorder(borderRadius: BorderRadius.circular(10.0)),
+                margin: const EdgeInsets.all(20.0),
+                color: Colors.lightBlue,
+            child: ListTile(
+                  onTap: (){
+                    print("Email Tapped");
+                  },
+              title: Text(_account.email, style: TextStyle(color: Colors.white, fontWeight: FontWeight.w500,),),
+                )
+            ),
+
+            Center(
+              child: Text(
+                "First Name",
+                style: TextStyle(
+                  fontSize: 18,
+                  fontWeight: FontWeight.bold,
+                ),
+                textAlign: TextAlign.center,
+              ),
+            ),
+
+          Card(
+            shape: RoundedRectangleBorder(borderRadius: BorderRadius.circular(10.0)),
+            margin: const EdgeInsets.all(20.0),
+            color: Colors.lightBlue,
+//            child: ListTile(
+             child: TextFormField(
```

Comment:
If you do decide to keep the cards, the edges of the text fields hang over the edges of the cards